infra/ansible/config/apt.yaml

```diff
@@ -13,7 +13,7 @@ apt:
       - rename
       - vim
       - wget
-      - clang-format-7
+      - clang-format
       - lcov
```

Comment:
can we add gcc-10 here directly?

Reply:
gcc-10 is already included in the build stage for `aarch64`. Maybe just move it to `build_common`?

Reply:
done.